SETUP.md

```diff
@@ -57,6 +57,16 @@ To perform backup, BBQ need rights to read BigQuery data from project which shou
 * Grant this permission through Google Cloud Console in [IAM tab](https://console.cloud.google.com/iam-admin/iam) for project which should be backed up. 
 * Grant this permission for the whole folder or organisation. It will be inherited by all of the projects underneath.
 
+### Cloud Datastore export
+  BBQ may export periodically data from Datastore to Big Query. Holding them additionally in Big Query insures against accidental deletion and allows to browse data about tables and its backups in the easier way. To turn it on:
```

Comment:
I'm not sure if we should say that export works as a backup because we've never tried the reverse operation. I think native datastore backups is better protection from accidental data loss.

I guess main motivation for export is easier search and browing.

Reply:
Done